UnityProject/Assets/Scripts/Items/Bureaucracy/Megaphone.cs

```diff
@@ -34,7 +34,14 @@ private IEnumerator Cooldown()
 
 		public bool RunChecks()
 		{
-			return isOnCooldown == false && isOn == true && pickupable.ItemSlot != null;
+			return IsOnCooldown() && isOn == true && pickupable.ItemSlot != null;
+		}
+
+		private bool IsOnCooldown()
+		{
+			if (isEmmaged) return false;
+			if(isOnCooldown && pickupable.ItemSlot != null && pickupable.ItemSlot.Player != null) Chat.AddExamineMsg(pickupable.ItemSlot.Player.gameObject,"The megaphone is on cooldown..");
```

Comment:
Could we rephrase this to be more "diegetic"? How about,
```suggestion
			if (isOnCooldown && pickupable.ItemSlot != null && pickupable.ItemSlot.Player != null)
			{
				Chat.AddExamineMsg(pickupable.ItemSlot.Player.gameObject, "The megaphone's internal capacitor is still recharging...");
			}
```
?